test/bdd/fixtures/vault-server/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
+
+services:
+
+  vault.server.example.com:
+    container_name: vault.server.example.com
+    image: ${VAULT_SERVER_IMAGE}:latest
+    environment:
+      - VAULT_HOST_URL=${VAULT_SERVER_HOST}:${VAULT_SERVER_PORT}
+      - VAULT_REMOTE_KMS_URL=https://vault.kms.example.com:${KMS_PORT}
+      - VAULT_EDV_URL=http://vault.edv.example.com:${EDV_PORT}/encrypted-data-vaults
+    ports:
+      - ${VAULT_SERVER_PORT}:${VAULT_SERVER_PORT}
+    entrypoint: ""
+    command: /bin/sh -c "cp /etc/tls/* /usr/local/share/ca-certificates/;update-ca-certificates; vault-server start"
+    volumes:
+      - ../keys/tls:/etc/tls
+    networks:
+      - couchdb_bdd_net
+
+  vault.kms.example.com:
+    container_name: vault.kms.example.com
+    image: ${KMS_IMAGE}:${KMS_IMAGE_TAG}
+    environment:
+      - KMS_HOST_URL=0.0.0.0:${KMS_PORT}
+      - KMS_DATABASE_TYPE=mem
+      - KMS_PRIMARY_KEY_DATABASE_TYPE=mem
+      - KMS_LOCAL_KMS_DATABASE_TYPE=mem
+      - KMS_KEY_MANAGER_STORAGE_TYPE=mem
+      - KMS_ZCAP_ENABLE=true
+      - KMS_TLS_SYSTEMCERTPOOL=true
+      - KMS_TLS_CACERTS=/etc/tls/ec-cacert.pem
+      - KMS_TLS_SERVE_CERT=/etc/tls/ec-pubCert.pem
+      - KMS_TLS_SERVE_KEY=/etc/tls/ec-key.pem
+    ports:
+      - ${KMS_PORT}:${KMS_PORT}
+    entrypoint: ""
+    command: /bin/sh -c "kms-rest start"
+    volumes:
+      - ../keys/tls:/etc/tls
+    networks:
+      - couchdb_bdd_net
+
+  vault.edv.example.com:
```

Comment:
we already have edv https://github.com/trustbloc/edge-service/tree/main/test/bdd/fixtures/edv-rest